app/views/requests/index.html.erb

```diff
@@ -6,10 +6,12 @@
   <thead>
     <tr>
       <th>Name</th>
-      <th>Cpu-Cores</th>
-      <th>MB RAM</th>
-      <th>MB Storage</th>
+      <th style="width: 80px">Cpu-Cores</th>
+      <th style="width: 80px">MB RAM</th>
+      <th style="width: 90px">MB Storage</th>
       <th>Operating system</th>
+      <th style="width: 80px">Port</th>
```

Comment:
Why did you add a width for those columns? :-)

Reply:
Because of the layout. It's look better with the width ;)
But I can delete it if you think it is to "hardcoded".

Reply:
Maybe we could/should use something like this: https://stackoverflow.com/questions/18503849/set-min-width-in-html-tables-td